### PR TITLE
[TIDOC-2628] Added missing responseHeaders property

### PR DESCRIPTION
**Pull request by Topener**
Ticket: https://jira.appcelerator.org/browse/TIDOC-2628

### CODE REVIEW

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 1]
```diff
@@ -192,6 +192,13 @@ methods:
         summary: Name of the header to retrieve.
         type: String
 
+  - name: getResponseHeaders
+    summary: Returns all response headers available for the request
+    returns:
+        type: Array
+    platforms: [iphone,ipad]
+
+
```
Comment (hansemannn) [inline apidoc/Titanium/Network/HTTPClient.yml]: Remove this block, it's generated automatically

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 2]
```diff
@@ -311,6 +318,16 @@ properties:
     permission: read-only
     platforms: [android]
     
+  - name: responseHeaders
+    summary: All of the response headers.
```
Comment (hansemannn) [inline apidoc/Titanium/Network/HTTPClient.yml]: Bad wording, please use a descriptive text.

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 3]
```diff
@@ -311,6 +318,16 @@ properties:
...
+    description: |
+        Contains an array of response headers
```
Comment (hansemannn) [inline apidoc/Titanium/Network/HTTPClient.yml]: It is a dictionary no area. Also already mentioned in the summary.

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 4]
```diff
@@ -311,6 +318,16 @@ properties:
...
+
+        See also: [getResponseHeader](Titanium.Network.HTTPClient.getResponseHeader).
+    type: Array
```
Comment (hansemannn) [inline apidoc/Titanium/Network/HTTPClient.yml]: Incorrect type, it is a dictionary.

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 5]
```diff
@@ -311,6 +318,16 @@ properties:
...
+    permission: read-only
+    platforms: [ios]
```
Comment (hansemannn) [inline apidoc/Titanium/Network/HTTPClient.yml]: Incorrect platform, it is `[iphone, ipad]`.

**Review by hansemannn — CHANGES_REQUESTED** [review]
We cannot merge this currently. Please address the comments before.

**Comment by Topener** [comment]
I've updated the changes

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 6]
```diff
@@ -311,6 +311,16 @@ properties:
     permission: read-only
     platforms: [android]
     
+  - name: responseHeaders
+    summary: Returns all the response headers returned with the request
+    description: |
+        Contains a dictionary of responseheaders only after the http call has finished.
+
+        See also: [getResponseHeader](Titanium.Network.HTTPClient.getResponseHeader).
+    type: Array<Object>
```
Comment (hansemannn) [inline apidoc/Titanium/Network/HTTPClient.yml]: The type still looks invalid. The code says it's a Dictionary [here](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiNetworkHTTPClientProxy.m#L536) and [here](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiNetworkHTTPClientProxy.h#L46). So it has to be: `type: Dictionary` unless you really receive an array.

**Review by hansemannn — CHANGES_REQUESTED** [review]

**Comment by hansemannn** [comment]
Applied the fix manually, PR approved!

**Review by hansemannn — APPROVED** [review]

**Pull request merged**